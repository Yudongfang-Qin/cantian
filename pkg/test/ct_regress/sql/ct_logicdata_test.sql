CREATE USER logic_u1 IDENTIFIED by Cantian123;
GRANT CREATE SESSION TO logic_u1 WITH ADMIN OPTION;
GRANT CREATE TABLE TO logic_u1;
GRANT DBA TO logic_u1;
COMMIT;

conn logic_u1/Cantian123@127.0.0.1:1611
ALTER DATABASE ENABLE_LOGIC_REPLICATION ON;
CREATE TABLE logic_t1 (id int primary key);
ALTER TABLE logic_t1 ADD LOGICAL LOG(PRIMARY KEY);
INSERT INTO logic_t1 VALUES(1);
DELETE logic_t1 WHERE id=1;
COMMIT;
