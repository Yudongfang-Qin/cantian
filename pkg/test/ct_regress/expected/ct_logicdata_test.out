

SQL> CREATE USER logic_u1 IDENTIFIED by Cantian123;

Succeed.

SQL> GRANT CREATE SESSION TO logic_u1 WITH ADMIN OPTION;

Succeed.

SQL> GRANT CREATE TABLE TO logic_u1;

Succeed.

SQL> GRANT DBA TO logic_u1;

Succeed.

SQL> COMMIT;

Succeed.

SQL> 
SQL> conn logic_u1/Cantian123@127.0.0.1:1611

connected.

SQL> ALTER DATABASE ENABLE_LOGIC_REPLICATION ON;

Succeed.

SQL> CREATE TABLE logic_t1 (id int primary key);

Succeed.

SQL> ALTER TABLE logic_t1 ADD LOGICAL LOG (PRIMARY KEY);

Succeed.

SQL> INSERT INTO logic_t1 VALUES(1);

1 rows affected.

SQL> DELETE logic_t1 WHERE id=1;

1 rows affected.

SQL> COMMIT;

Succeed.

SQL> 


